test/integration/auth/rbac_test.go

```diff
@@ -1037,3 +1040,73 @@ func TestRBACContextContamination(t *testing.T) {
 
 	}
 }
+
+func TestMonitoringURLs(t *testing.T) {
+	tests := []struct {
+		name        string
+		requestPath []string
+	}{
+		{
+			name: "monitoring endpoints",
+			requestPath: []string{
```

Comment:
can we add some regexp(s) here? that we can then use to test later when we get a http.StatusOK (line 1106)

Reply:
Do you mean regexp for the paths? Like having tests to test paths like "/healthz/etcd" or other subpaths?

Reply:
Clarified on slack, added some regexp matching for response body now.